src/Microsoft.ML.AutoML/CodeGen/search-space-schema.json

```diff
@@ -235,7 +236,8 @@
         "anchor",
         "dnnModelFactory",
         "bertArchitecture",
-        "imageClassificationArchType"
+        "imageClassificationArchType",
+        "DataKind"
```

Comment:
I do not think this is the right place @LittleLittleCloud   could you take look it ?

Reply:
please use small 'd'.
`dataKind`